### PR TITLE
Added API documentation

### LINKED ISSUE #5: Document API
Provide clear documentation for what the API does and how to use it

### PR DESCRIPTION
**Pull request by gramliu**
Added more detailed documentation about the inputs and outputs of the API. Resolves #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,15 +48,18 @@ $ curl \
...
+The output is a scalar value: `0` or `1`, which corresponds to whether or not a student is predicted to be successful by the end of the year, as measured by their final grade. A value of `1` means that a student is predicted to be successful whereas `0` means otherwise.
+
 ## Model Description
 
 ### Input Features
 The features we used in training our model are `health`, `absences`, `age`, and `G2`.
 
 ### Improvement from the Baseline Model
-By taking in G2 as an additional feature, our retrained model achieves an f1 score of 0.9861111111111112 on the test set. In comparison, the f1 score of the baseline model on the test set is 0.5185185185185185.
+By taking in G2 as an additional feature, our retrained model achieves an f1 score of `0.986 `on the test set. In comparison, the f1 score of the baseline model on the test set is `0.519`.
```
Comment (jfrucht25) [inline README.md]: I don't think this metric result was from the test set.

File: README.md [hunk 2]
```diff
@@ -37,15 +48,18 @@ $ curl \
...
 
 Thus, our retrained model has a significantly better out-of-sample accuracy than the baseline model, which indicates better model quality.
 
 ## Testing
-**(what testing have we done? How do we run tests?)**
+**(what testing have we done? How do we run tests?)**
```
Comment (jfrucht25) [inline README.md]: tests can be run by running pytest in the parent directory. They require that the container be running. (I can add the testing info if you want).

File: README.md (hunk 1 above)
Comment (jfrucht25) [reply README.md]: The original algorithm had no cross-validation, so it only measured based on the training set

**Review by jfrucht25 — CHANGES_REQUESTED** [review]
Most looks fine, just some small details that should be double checked

File: README.md (hunk 1 above)
Comment (gramliu) [reply README.md]: Oh. In that case, should we run additional metrics so that the two models are benchmarked against the same dataset? @thisisalbertliang

File: README.md (hunk 2 above)
Comment (gramliu) [reply README.md]: If you could add this, that would be great. Since you wrote the tests, I think that you'd probably know how to best explain this part

File: README.md (hunk 1 above)
Comment (gramliu) [reply README.md]: After looking at the two notebooks in more detail, it doesn't seem like we're splitting the datasets at all across test and train. I think that's something we should probably incorporate to make sure that we're not overfitting on the train dataset.

Comment (jfrucht25) [reply README.md]: I added cross-validation in #9 but it hasn't been merged

**Comment by shawnchan2014** [comment]
Looks good!
